docs/development.rst

```diff
@@ -505,6 +505,357 @@ point, with diagrams, and an alternative workflow, see `the git docs
+mentioned above from the literature.
+You will later need to specify these citations in your code files
+(see `Adding and editing the required files`_).
+
+
+-----------------------------------
+Getting set up to add a new species
+-----------------------------------
+
+If this is your first time adding a species to `stdpopsim`, it's a good
+idea to take some time browsing the :ref:`sec_catalog`
+to see how existing species are typically specified and documented. If you have
+any questions or confusion about the required code, please
+don't hesitate to open an `issue <http://github.com/popgensims/stdpopsim/issues>`_.
+We're more than happy to answer any questions and help get you up and running.
+Before you add any code, be sure to have forked the `stdpopsim` repository
+and cloned it locally, following the instructions in the `GitHub Workflow`_ section.
+After you collected the relevant parameters from the literature (see list above),
+the first step is to create a new subdirectory inside the `catalog/`
+directory named for the species (see `Naming conventions`_ for more details).
+All code described below should go in this directory unless explicitly specified otherwise.
+
+-------------------------------------
+Adding and editing the required files
+-------------------------------------
+
+`stdpopsim` has an automated procedure for generating template files where you can enter
+all relevant information for your species.
+These files can be automatically initialized with data pulled down from Ensembl.
+To do this, execute the `maintenance` command with a "_" delimited Ensembl species ID.
+A partial list of the
+genomes housed on Ensembl can be found `here <https://metazoa.ensembl.org/species.html>`__.
+For example, the template files for *A. thaliana* were generated by executing this command:
+
+.. code-block:: shell
+
+    python -m maintenance add-species arabidopsis_thaliana
+
+The `maintenance` utility generates three new files inside the `catalog/{species_id}/` directory:
+
+* `__init__.py`
+* `genome_data.py`
+* `species.py`
+
+The `__init__.py` file is a wrapper script that loads all the relevant information for your species. It should be edited only when you add components to your species, such as demographic models,
+genetic maps, or DFEs.
+
+The `genome_data.py` file contains the physical map of the genome.
+The `maintenance` utility generates this file autmatically
+based on information it downloaded from Ensembl.
+Essentially, this utility sucks down a whole lot of useful information for free.
+The `genome_data.py` file contains a data dictionary which has slots for the
+assembly accession number, the assembly name,
+and a dictionary representing the chromosome names and their associated lengths.
+If synonyms are defined (e.g., chr2L for 2L) then those are given in the list that follows.
+You should double-check the downloaded values, but there is probably no reason to edit this file.
+
+The `species.py` file
```

Comment:
This looks like it should be a header of some kind

Reply:
Actually not. I didn't want to have too many short subsections. The sentence just got cut. I changed it to:
> The `species.py` file is where you specify the mutation and recombination rates, effective population size, and the average generation time, along with all accompanying citations (see details in **Which information do I need to have for my species?**).

Reply:
I restructured this a bit now. See latest commit.